ios/FastImage/FFFastImageView.m

```diff
@@ -12,6 +14,27 @@ - (void)setResizeMode:(RCTResizeMode)resizeMode
     }
 }
 
+- (void)setOnFastImageLoadEnd:(RCTBubblingEventBlock)onFastImageLoadEnd {
```

Comment:
I would have like a different commit for ios stuff. It is easier to read as a reviewer.

Reply:
Ok, will do next PR. Sorry for that.